pom.client.xml

```diff
@@ -938,6 +938,7 @@
     <module>sdk/core/azure-core-http-okhttp</module>
     <module>sdk/core/azure-core-test</module>
     <module>sdk/core/azure-core-tracing-opencensus</module>
```

Comment:
Are we going to sunset/deprecate the OpenCensus module?

Reply:
Yeah, Once we have a stable version for Opentelemetry we can remove the opencensus module.